include/window.h

```diff
@@ -1,6 +1,13 @@
 #ifndef GUARD_WINDOW_H
 #define GUARD_WINDOW_H
 
+// The tile ID for a transparent tile in the window tilesheet
+#define TRANSPARENT_TILE_NUMBER 0
+
+#define PIXEL_BUFFER_TRANSPARENT 0x00
+#define PIXEL_BUFFER_WHITE 0x11
```

Comment:
You shouldn't make the assumption this will always be the white color. Any color could be loaded here. I don't even think these need constants

Reply:
I named it because it's specifically the menu's palette. Clear-this-area-to-white happens extremely often in the message box code.

Reply:
I ended up making a new macro for this that turned out to be much more useful.